batch/deployment.yaml

```diff
@@ -10,6 +10,7 @@ metadata:
   labels:
     app: batch
     hail.is/sha: "{{ code.sha }}"
+    grafanak8sapp: "true"
```

Comment:
Is this how booleans should be in a yaml file? https://yaml.org/type/bool.html

Reply:
Actually, maybe this is fine and it needs to be a string

Reply:
This is how it is in other places: https://github.com/hail-is/hail/blob/master/monitoring/monitoring.yaml#L21

It has to be a string I think because we search it with a regex here: https://github.com/hail-is/hail/blob/master/monitoring/monitoring.yaml#L282